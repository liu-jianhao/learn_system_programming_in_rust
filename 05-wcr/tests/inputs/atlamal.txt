Frétt hefir öld óvu, þá er endr of gerðu
seggir samkundu, sú var nýt fæstum,
æxtu einmæli, yggr var þeim síðan
ok it sama sonum Gjúka, er váru sannráðnir.
