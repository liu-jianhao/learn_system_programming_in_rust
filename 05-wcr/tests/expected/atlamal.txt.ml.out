       4     159 tests/inputs/atlamal.txt
